src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Ball {
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    private String color;
+    private int number;
+}
```

Comment:
Consider providing a constructor for the Ball class to initialize the 'color' and 'number' fields. This can make object creation more convenient and less error-prone.